### PR TITLE
Cybits Network Implementation

### PR DESCRIPTION
**Pull request by CybitsBlockchain**

### CODE REVIEW

File: src/BlockCore.Networks.Cybits/Networks/Policies/CybitsStandardScriptsRegistry.cs [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+namespace Blockcore.Networks.Cybits.Policies
+{
+   /// <summary>
+   /// Blockcore sample coin-specific standard transaction definitions.
+   /// </summary>
+   public class CybitsStandardScriptsRegistry : StandardScriptsRegistry
+   {
+      // See MAX_OP_RETURN_RELAY in stratisX, <script.h>
+      public const int MaxOpReturnRelay = 40;
```
Comment (dangershony) [inline src/BlockCore.Networks.Cybits/Networks/Policies/CybitsStandardScriptsRegistry.cs:16]: this should probably be bigger maybe like in bitcoin 82 bytes

**Review by dangershony — APPROVED** [review]
apart for one comment looks good to push

File: src/BlockCore.Networks.Cybits/Networks/Policies/CybitsStandardScriptsRegistry.cs (hunk 1 above)
Comment (CybitsBlockchain) [reply src/BlockCore.Networks.Cybits/Networks/Policies/CybitsStandardScriptsRegistry.cs]: updated to 83

**Review by dangershony — APPROVED** [review]
good to push

**Pull request merged**